ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/QueryDescription.java

```diff
@@ -130,7 +145,8 @@ public boolean equals(final Object o) {
         && Objects.equals(sources, that.sources)
         && Objects.equals(sinks, that.sinks)
         && Objects.equals(overriddenProperties, that.overriddenProperties)
-        && Objects.equals(state, that.state);
+        && Objects.equals(state, that.state)
+        && Objects.equals(ksqlHostQueryState, that.ksqlHostQueryState);
```

Comment:
Consider adding a unit test using `EqualsTester` if the class does not already have one.